cloud/pkg/controllermanager/controllermanager.go

```diff
@@ -66,6 +66,8 @@ func setupControllers(ctx context.Context, mgr manager.Manager) error {
 				&overridemanager.ReplicasOverrider{},
 				&overridemanager.ImageOverrider{},
 				&overridemanager.NodeSelectorOverrider{},
+				&overridemanager.CommandOverrider{},
+				&overridemanager.ArgsOverrider{},
```

Comment:
Only commandoverrider and argoverriders  implemented in the PR temporarily? Is envoverriders and resourceoverriders only added in the API definition?

Reply:
Yes, because the amount of code is relatively large, it is divided into two commits, and the implementation of envoverriders and resourceoverriders will be submitted next time.